src/actions/index.js

```diff
@@ -62,9 +62,10 @@ export const RESET_ALL_STATE = 'RESET_ALL_STATE';
 //
 // Action Creators
 //
-export const addProject = (project: Project) => ({
+export const addProject = (project: Project, onboardingCompleted: boolean) => ({
```

Comment:
Nit: Sorry I didn't think of this sooner, but how do you feel about changing this to `isOnboardingCompleted`, both here and everywhere else as well?

I like using `isX` for boolean variables as a quick way to tell whether the variable is a boolean. Plus then it'd be consistent with others (like `isVisible` below)

Reply:
Yeah I like how that sounds, kinda like a question with a yes answer, cause we'll only ever have 'true' as the value.

Reply:
btw, not related to this PR but the type of `project` is wrong I believe. It should be `InternalProject` since it's in the reducer that it's transformed into a `Project`. Same for `importExistingProjectFinish`